src/main/plugins/org.dita.xhtml/xsl/dita2html-base_template.xsl

```diff
@@ -28,6 +28,9 @@ See the accompanying LICENSE file for applicable license.
 
 <!-- the dita to xhtml converter for task documents -->
 <xsl:import href="xslhtml/taskdisplay.xsl"/>
+  
+  <!-- the dita to xhtml converter for troubleshooting documents -->
+  <xsl:import href="xslhtml/troubleshootingdisplay.xsl"/>
```

Comment:
This is only changing XHTML; I think we'll want equivalent changes in HTML5. The two have now branched, so once you've got the troubleshooting override finalized for one of them, you should copy it into the other and make equivalent updates in the other template.